framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3494,6 +3524,8 @@ public AnnotatedExecutableType getFunctionTypeFromTree(LambdaExpressionTree tree
         return Pair.of(functionalInterfaceType, functionType);
     }
 
+    //    void getFnInterfaceFromTreePreSubstitution()
```

Comment:
This is no longer needed? Or is there still something to explore for functional interfaces?